pkg/generate/generate.go

```diff
@@ -269,14 +270,19 @@ func applyRule(log logr.Logger, client *dclient.Client, rule kyverno.Rule, resou
 		logger.V(4).Info("created new resource")
 
 	} else if mode == Update {
-		logger.V(4).Info("updating existing resource")
-		// Update the resource
-		_, err := client.UpdateResource(genKind, genNamespace, newResource, false)
-		if err != nil {
-			// Failed to update resource
-			return noGenResource, err
+		if rule.Generation.Synchronize {
```

Comment:
Is `Synchronize` disabled for the pre-existing generate policy? Can we update this behavior in the document?

Reply:
ok